src/components/hero/campaignHeroSubText.tsx

```diff
@@ -36,7 +36,14 @@ const CampaignHeroSubText = ({ featuredItem }: any) => {
       </Text>
       <HorizontalRule />
       <Box>
-        <Text color="ui.typography.body" mb="0px" noOfLines={2}>
+        <Text
+          color="ui.typography.body"
+          mb="0px"
+          noOfLines={2}
+          __css={{
+            ":focus-within": { overflow: "visible" },
```

Comment:
Necessary bc of Chakra's otherwise super useful `noOfLines` -> `lineClamp` for this link

Reply:
I wonder if v3 of the DS would affect this. I won't want this fix to be mixed with the DS v3 update but it'd be worth checking in a separate branch if v3 changes this (because of the Chakra update).

Reply:
Checked it out and it didn't seem to fix it out of the box. I guess that makes sense bc the `noOfLines` prop is working as expected here, it's just a blunt instrument